src/recovery/mod.rs

```diff
@@ -914,7 +914,7 @@ impl Recovery {
     }
 
     #[cfg(feature = "qlog")]
-    pub fn maybe_qlog(&mut self) -> Option<qlog::event::Event> {
+    pub fn maybe_qlog(&mut self) -> Option<qlog::EventData> {
```

Comment:
Same, why not return `qlog::Event` directly? Would save having to call `qlog::Event::new` from the caller.

Reply:
same answer :)